types/koa-session/koa-session-tests.ts

```diff
@@ -0,0 +1,8 @@
+import * as Koa from 'koa';
+import * as session from 'koa-session';

```

Comment:
@minestarks thank for your suggestion.
I have modified it to be common js module.
I'm reference from the definition file of koa and express.
They are all not true es6 module but their test file also using  es6 module style.